draft-homburg-deleg-incremental-deleg.md

```diff
@@ -0,0 +1,607 @@
+Legacy methods to redirect this information to another location, possible under the control of another operator, such as (CNAME {{Section 3.6.2 of RFC1034}}) and DNAME {{!RFC6672}} remain functional.
+One could even outsource all delegation operational practice to another party with an DNAME on the `_deleg` label on the apex of the delegating zone.
+
+Additional to the legacy methods, a delegation may be outsourced to a third party by having an SVCB RRset with a single SVCB RR in AliasMode.
+
+## DNSSEC protection of the delegation {#dnssec-protection}
+
+With legacy delegations, the NS RRset at the parent side of a delegation as well as glue records for the names in the NS RRset are not authoritative and not DNSSEC signed.
+An adversary that is able to spoof a referral response, can alter this information and redirect all traffic for the delegation to a rogue name server undetected.
+The adversary can then perceive all queries for the redirected zone (Privacy concern) and alter all unsigned parts of responses (such as further referrals, which is a Security concern).
+
+DNSSEC protection of delegation information prevents that, and is the only counter measure that also works against on-path attackers.
+At the time of writing, the only way to DNSSEC validate and verify delegations at all levels in the DNS hierarchy is to revalidate delegations {{?I-D.ietf-dnsop-ns-revalidation}}, which is done after the fact and has other security concerns ({{Section 7 of ?I-D.ietf-dnsop-ns-revalidation}}).
+
+Direct delegation information (provided by SVCB RRs in ServiceMode) include the hostnames of the authoritative name serversfor the delegation as well as IP addresses for those hostnames.
+Since the information is stored authoritatively in the delegating zone, it will be DNSSEC signed if the zone is signed.
+When the delegation is outsourced, then the delegation is protected when the zones providing the aliasing resource records *and* the zones serving the targets of the aliases are all DNSSEC signed.
+
+## Maximize ease of deployment {#deployability}
+
+Delegation information is stored authoritatively within the delegation zone.
+No semantic changes as to what zones are authoritative for what data are needed.
+As a consequence, existing DNS software, such as authoritative name servers and DNSSEC signing software, can remain unmodified.
+Unmodified authoritative name server software will serve the delegation information when queried for.
+Unmodified signers will sign the delegation information in the delegating zone.
+Only the recursive resolver needs modification to follow referrals as provided by the delegation information.
+
+Such a resolver would explicitly query for the delegations administered as specified in {{delegation-administration}}.
+The number of round trips form the recursive resolver to the authoritative name server is comparable to what is needed for DNS Query Name Minimisation {{!RFC9156}}.
```

Comment:
s/form/from/

Reply:
Also I feel like comparable implies that it can differ, but im pretty sure it is exactly the same amount of round trips